### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rgreinho**
Simplifies the README to quickly provide the information to start the
project as well as pointing the user to other resources ifthey want to
dig deeper.

Drive-bys:
* Adds a CONTRIBUTING.md into the .github folder to improve the GitHUB
  integration.
* Adds a setup target to the Makefile to fully deploy the SSF in one
  command.

Fixes thesecuresoftwarefactory/ssf#16

### CODE REVIEW

**Comment by mlieberman85** [comment]
I want to hold off on the contribution stuff for a little bit. Just want to make sure we have a few things cleared. Not sure what code of conduct and whatever else we're doing. It might not be left up to us.

File: Makefile [hunk 1]
```diff
@@ -21,6 +21,9 @@ help: # Display help
 			printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
 		}' $(MAKEFILE_LIST) | sort
 
+.PHONY: setup
+setup: setup-minikube setup-tekton-chains tekton-generate-keys setup-kyverno example-buildpacks ## Sets up the SSF project into minikube
```
Comment (bradbeck) [inline Makefile:25]: This does a little bit more than just "setup". It actually triggers a pipeline run.

**Review by bradbeck — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (rgreinho) [reply Makefile:25]: Ah yup, that's true. 🤔 Any idea/suggestion to rename it to something else?

Comment (bradbeck) [reply Makefile:25]: Quickstart?

Comment (rgreinho) [reply Makefile:25]: Haha, straightforward! I like that 👍

**Review by mlieberman85 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+template = "docs/page.html"
++++
+
+The SSF project welcomes any kind of contributions, from code to documentation
+via fixing typos. Please feel free to raise an [issue] if you would like to
+work on something major to ensure efficient collaboration and avoid duplicate
+effort.
+
+The project follows the
+[CNCf Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
```
[the remark this answers is not in the document]
Comment (rgreinho) [reply .github/CONTRIBUTING.md:18]: Addressed ✅

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  [How to Write a Git Commit Message] for more details.
+* If your [pull request] is a work in progress, create it as a
+  [draft pull request].
+* Any [pull request] inactive for 28 days will be automatically closed. If you
+  need more time to work on it, ask maintainers, to add the appropriate label to
+  it. Use the `@` mention in the comments.
+* Unless explicitly asked, [pull request] which don't pass all the CI checks
+  will not be reviewed. Use the `@` mention in the comments to ask maintainers
+  to help you.
+
```
[the remark this answers is not in the document]
Comment (rgreinho) [reply .github/CONTRIBUTING.md:44]: Addressed ✅

File: README.md [hunk 4]
```diff
@@ -1,62 +1,46 @@
-> **NOTE: See [old README](README.old.md) for out of date info that provides some context while the Secure Software Factory is being rearchitected**
-
 # The Secure Software Factory
```
[the remark this answers is not in the document]
Comment (rgreinho) [reply README.md]: Addressed ✅

**Comment by rgreinho** [comment]
Alright @mlieberman85 I think I addressed all the concerns! Let me know what you think.

**Review by mlieberman85 — DISMISSED** [review]

File: Makefile [hunk 5]
```diff
@@ -21,6 +21,9 @@ help: # Display help
 			printf "\033[36m%-30s\033[0m %s\n", $$1, $$NF \
 		}' $(MAKEFILE_LIST) | sort
 
+.PHONY: quickstart
+setup: setup-minikube setup-tekton-chains tekton-generate-keys setup-kyverno example-buildpacks ## Spin up the SSF project into minikube
```
Comment (bradbeck) [inline Makefile:25]:
```suggestion
quickstart: setup-minikube setup-tekton-chains tekton-generate-keys setup-kyverno example-buildpacks ## Spin up the SSF project into minikube
```

**Review by bradbeck — APPROVED** [review]

**Review by mlieberman85 — APPROVED** [review]

**Pull request merged**